docs/form-guide.md

```diff
@@ -6,54 +6,48 @@ We use [`react-hook-form`](https://react-hook-form.com/) and [`zod`](https://git
 
 ### 1. Use `zod` to create form field schema
 
+You can set error message with i18n, you can read this file to learn about [i18n](i18n-guide.md)
+
 ```js
 import { z as zod } from 'zod'
-
+const t = useI18n() // Set errror messge with i18n.
```

Comment:
stop, schema definition may not be react-hook

Reply:
🤔  Then perhaps i18n support will not be available

Reply:
We can create the schema inside a component (might memoize with useMemo), then we can use the hook.

Reply:
> We can create the schema inside a component (might memoize with useMemo), then we can use the hook.

Yes, that's how it works in the demo.